### DISCUSSION TITLE
Q on AS-20 Kayak guidance

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#4371] [Q&A] [answered]
In Icebreakers is the SSC-6 Sennit with the AS-20 Kayak missile. The missile seems to have on Sensors: Active Radar Seeker and a short range Passive Radar Seeker. Does this means the active missile has a ARM component?
regards GJ

**Comment by PygmalionOfCyprus** [comment, marked as answer]
I believe those are there to enable its Home-On-Jam capability.

**Comment by Parel803** [comment]
Rgr, thx I understand. Needed in the game for sim.
Thx for your time and answer.
regards GJ
